packages/desktop-client/src/components/mobile/budget/BudgetTable.jsx

```diff
@@ -1294,6 +1282,388 @@ export function BudgetTable({
   );
 }
 
+function Banner({ type = 'info', children }) {
+  return (
+    <Card
+      style={{
+        height: 50,
+        marginTop: 10,
+        marginBottom: 10,
+        padding: 10,
+        justifyContent: 'center',
+        backgroundColor:
+          type === 'critical'
+            ? theme.errorBackground
+            : type === 'warning'
+              ? theme.warningBackground
+              : theme.noticeBackground,
+      }}
+    >
+      {children}
+    </Card>
+  );
+}
+
+function UncategorizedTransactionsBanner(props) {
+  const { t } = useTranslation();
+  const count = useSheetValue(uncategorizedCount());
+  const navigate = useNavigate();
+
+  if (count === null || count <= 0) {
+    return null;
+  }
+
+  return (
+    <GridListItem textValue="Uncategorized transactions banner" {...props}>
+      <Banner type="warning">
+        <View
+          style={{
+            flex: 1,
+            flexDirection: 'row',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+          }}
+        >
+          {t('You have {{count}} uncategorized {{transactionText}}', {
+            count,
+            transactionText: count === 1 ? t('transaction') : t('transactions'),
+          })}
```

Comment:
<img width="902" alt="image" src="https://github.com/user-attachments/assets/080dc3d4-8a10-4aa1-993e-b6ad9f7d7758" />

```suggestion
          <Trans>You have {{count}} uncategorized transactions</Trans>
```

We should be able to handle the pluralisation through weblate (maintainers only I believe because editing source is disabled)